src/main/java/com/app/models/dto/UserDto.java

```diff
@@ -0,0 +1,4 @@
+package com.app.models.dto;
```

Comment:
```suggestion
package com.app.dtos;
```
ri dc, ko cần bỏ dto trong models làm j nhá